components/RichBody/index.tsx

```diff
@@ -20,36 +20,60 @@ interface Props {
   classes?: string; // custom styles for regular text (color, font weight, etc.)
 }
 
+interface DirectionObject {
```

Comment:
This variable name probably needs more context. I suggest calling it `NodeWithTextDirection`.